### PR TITLE
add python api to readthedocs

### PR DESCRIPTION
**Pull request by pinggao187**
test [#3501](https://github.com/intel-analytics/analytics-zoo/pull/3501)
https://zoo-readthedocs.readthedocs.io/en/add_zoo/

### CODE REVIEW

File: readthedocs/source/index.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ Analytics Zoo includes the **Orca** library that seamlessly scale out your singl
    :maxdepth: 1
    :caption: API reference
```
Comment (jason-dai) [inline readthedocs/source/index.rst:53]: change to "Python API"

File: readthedocs/source/doc/PythonApi/Orca/zoo.orca.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
+Orca API Documentation
```
Comment (jason-dai) [inline readthedocs/source/doc/PythonApi/Orca/zoo.orca.rst:1]: Change to "Orca API"; rename the folder to PythonAPI

Comment (helenlly) [reply readthedocs/source/doc/PythonApi/Orca/zoo.orca.rst]: done

**Comment by helenlly** [comment]
LGTM

File: readthedocs/source/index.rst [hunk 3]
```diff
@@ -50,9 +50,9 @@ Analytics Zoo includes the **Orca** library that seamlessly scale out your singl
 
 .. toctree::
    :maxdepth: 1
-   :caption: API reference
+   :caption: Python API
    
-   doc/api/api.md
+   doc/PythonAPI/Orca/zoo.orca.rst
```
Comment (jason-dai) [inline readthedocs/source/index.rst:55]: just orca.rst

Comment (helenlly) [reply readthedocs/source/index.rst:55]: done

**Comment by jason-dai** [comment]
LGTM

**Pull request merged**